go.mod

```diff
@@ -113,6 +113,11 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1
 )
 
+require (
+	github.com/streamingfast/logging v0.0.0-20220405224725-2755dab2ce75 // indirect
+	go.mongodb.org/mongo-driver v1.15.0 // indirect
+)
```

Comment:
Stray `indirect` block

Reply:
`go mod tidy` is not deterministic. Nobody seems to understand why these separate blocks are created :shrug: 
I tried to write a tool tool to format a mod file, but couldn't find a good enough modfile parser at the time. Maybe should try again.